server/src/main/java/org/apache/druid/server/StatusResource.java

```diff
@@ -75,7 +78,29 @@ public Map<String, String> getProperties()
   {
     Map<String, String> allProperties = Maps.fromProperties(properties);
     Set<String> hidderProperties = druidServerConfig.getHiddenProperties();
-    return Maps.filterEntries(allProperties, (entry) -> !hidderProperties.contains(entry.getKey()));
+    return filterHiddenProperties(hidderProperties, allProperties);
+  }
+
+  /**
+   * filter out entries from allProperties with key containing elements in hidderProperties (case insensitive)
+   *
+   * @return map of properties that are not filtered out.
+   */
+  @Nonnull
+  private Map<String, String> filterHiddenProperties(
+      Set<String> hidderProperties,
```

Comment:
should this be `hiddenProperties` instead of `hidderProperties`?

Reply:
Sharp! Will refactor it